src/request.js

```diff
@@ -56,7 +56,7 @@ class Request {
       headers: {
         'Content-Type': 'application/json'
       },
-      body: opts.body ? JSON.stringify(opts.body) : ''
+      body: JSON.stringify(opts.body)
```

Comment:
Will we get an error when opts.body is empty as the condition is removed?

Reply:
Yeh . If the body is null or undefined, we will get an error.